Libplanet/Tx/TxId.cs

```diff
@@ -119,6 +119,7 @@ public ImmutableArray<byte> ByteArray
         [Pure]
         public override string ToString() => ToHex();
 
+        /// <inheritdoc cref="IComparable{T}.CompareTo(T)"/>
         public int CompareTo(TxId other)
```

Comment:
Huh, didn't know `TxId`'s implemented `IComparable<T>`. Is this still needed?

Reply:
It's used for ordering transactions by their ID … when blocks are serialized?

Reply:
Sorry, my bad. Found it. 😓